operators/advanced_network/python/check_system.py

```diff
@@ -198,9 +198,9 @@ def check_mrrs():
             mrrs_value = (int(mrrs_result.stdout.strip(), 16) & 0xF000) >> 12
 
             if mrrs_value == 5:
-                logging.info(f"{pci_address}: MRRS is correctly set to 4096.")
+                logging.info(f"{name}/{pci_address}: MRRS is correctly set to 4096.")
```

Comment:
gotta loop on the result of `get_nic_info`. `name` is a list too. current:

```
2024-12-09 11:42:46 - WARNING - ['cx7_0', 'cx7_1']/0005:03:00.0: MRRS is set to 4194304, not 4096.
```

Reply:
Thanks. Fixed